src/components/InputForm.module.css

```diff
@@ -0,0 +1,9 @@
+.InputForm {
+    background-color: #ffffff !important;
```

Comment:
As above, it's better to use tailwind utilities classes. you can use the `@apply` method to apply classes and use `!important` to override flatifycss. 

e.g: `.InputForm {
@apply bg-white shadow-none !important
}`